docs/_data/docstoc.yml

```diff
@@ -51,11 +51,11 @@
 
 - title: 'Using Codewind remotely'
   children:
-    - title: 'Overview'
+    - title: 'Overview: Using Codewind remotely'
       url: remote-jane-overview.html
     - title: 'Connecting your IDE to remote Codewind'
       children:
-      - title: 'Connecting your VS Code to remote Codewind'
+      - title: 'Connecting VS Code to remote Codewind'
```

Comment:
File name is the original version, so we'll have to go in and change that: https://github.com/eclipse/codewind-docs/blob/master/docs/_documentations/remotedeploy-vscode.md